MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-include *.py MANIFEST.in
-global-include *.txt *.rst *.ini *.yml *.cfg *.crt *.key
-global-exclude .gitignore
+graft aiosmtpd
+global-include *.txt *.rst *.ini *.yml *.cfg *.crt *.key LICENSE
```

Comment:
Since we have explicitly specified to grab the whole `aiosmtpd` directory, do we still need `global-include`?

Maybe replace this with a non-global `include` directive instead to grab the build/test files in the project's root?

Reply:
Fair point. Need to check this.